tools/cli/internal/breakingchanges/exemptions.go

```diff
@@ -57,7 +61,8 @@ func transformComponentEntry(breakingChangeDescription string) string {
 	return breakingChangeDescription
 }
 
-func getValidExemptionsList(exemptionsPath string, ignoreExpiration bool, fs afero.Fs) ([]Exemption, error) {
+// GetValidExemptionsList returns a list of exemptions. If ignoreExpiration is set to true, it will return all exemptions.
+func GetValidExemptionsList(exemptionsPath string, ignoreExpiration bool, fs afero.Fs) ([]Exemption, error) {
```

Comment:
I think we could use `os.Open` instead of afero since it doesn't bring any value using it here but we are passing it as input params from different functions

Reply:
i think it's a pattern i noticed in our commands, but I thinkwe should start changing to afero as we can then inject files in tests much easily and avoid file issues, let me know what you think i can revert back and change all in one pr but i think it will take time